### PR TITLE
nirmata/kubectl image

### PR DESCRIPTION
**Pull request by anushkamittal2001**

### CODE REVIEW

File: aws/values.yaml [hunk 1]
```diff
@@ -316,7 +316,7 @@ webhooksCleanup:
   # -- Create a helm pre-delete hook to cleanup webhooks.
   enabled: true
   # -- `kubectl` image to run commands for deleting webhooks.
-  image: bitnami/kubectl:latest
+  image: ghcr.io/nirmata/kubectl:1.29.0
```
Comment (pns-nirmata) [inline aws/values.yaml:319]: I think we need to point to the AWS registry right? Just like we have done for admission-controller etc?

**Review by pns-nirmata — APPROVED** [review]
Need to change to aws registry for images like we do for admission controller ones.

File: aws/values.yaml (hunk 1 above)
Comment (anushkamittal2001) [reply aws/values.yaml:319]: Right, done!

Comment (anushkamittal2001) [reply aws/values.yaml]: Right, done!

**Pull request merged**